ait/core/dmc.py

```diff
@@ -302,6 +304,10 @@ def _load_leap_second_data(self):
                     log.warn('Continuing with out of date leap second data')
                 else:
                     raise ValueError('Could not load leap second data')
+        else:
+            t = self._data['valid']
+            t.strftime('%m/%d/%Y')
+            log.info('Leapseconds data valid until %s', t)
```

Comment:
This is going to print the default format for `t`. Something like this:
```
>>> print(ait.core.dmc._data['valid'])
2020-12-28 00:00:00
```

You need to store the return value from `strftime` to change the format.